### PR TITLE
Fix some small issues in the getting started docs

### PR DESCRIPTION
**Pull request by tryantwit**
* Modified a reference to real time to match the previous line reference of realtime.
* Modified eg to e.g. as it's an abbreviation for the latin exempli gratia.
* Added missing pronoun to `_executing_filters` section to read better.

### CODE REVIEW

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ There are a few concepts that are core to Elasticsearch. Understanding these con
 [float]
 === Near Realtime (NRT)
 
-Elasticsearch is a near real time search platform. What this means is there is a slight latency (normally one second) from the time you index a document until the time it becomes searchable.
+Elasticsearch is a near realtime search platform. What this means is there is a slight latency (normally one second) from the time you index a document until the time it becomes searchable.
```
Comment (DaveCTurner) [inline docs/reference/getting-started.asciidoc]: I think I'd prefer `near-realtime`.

**Review by DaveCTurner — CHANGES_REQUESTED** [review]
Looks good, thanks @tryantwit. I asked for one minor change.

**Comment by karmi** [comment]
Hi @tryantwit, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/elasticsearch/pull/30346.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by tryantwit** [comment]
Hey @karmi, sorry about that. Got the other email added to github.

@DaveCTurner pull request was updated with your requested change. Though after doing a quick search through the rest of Elasticsearches documentation it look like the docs are usage the phrase `near real-time`.

Those references can be found in `docs/reference/search/suggesters/completion-suggest.asciidoc` and `docs/reference/modules/indicies/request_cache.asciidoc`. It'd probably be best to keep it consistent throughout the docs so I can modify my reference or modify those as well. Just let me know what you think.

**Review by DaveCTurner — APPROVED** [review]
LGTM

**Comment by DaveCTurner** [comment]
@elasticmachine test this please

**Comment by javanna** [comment]
retest this please

**Comment by DaveCTurner** [comment]
@elasticmachine test this please

**Comment by DaveCTurner** [comment]
@elasticmachine test this please

**Pull request merged**